cluster/ci/config/roles.yaml

```diff
@@ -274,3 +280,16 @@ items:
     kind: ClusterRole
     apiGroup: rbac.authorization.k8s.io
     name: system:image-puller
+- kind: RoleBinding
```

Comment:
Unnecessary, all accounts on the cluster can pull these secrets.

Reply:
The goal is to be able to pull images from the ocp namespace like 

> registry.svc.ci.openshift.org/ocp/release:4.0.0-0.nightly-2019-02-08-145216

@smarterclayton , are you saying that RoleBinding can be removed and this service account will still be able to pull images from `ocp` namespace? Not sure what secrets you mean.

Reply:
Ok, I removed this RoleBinding. Thank you.